openpype/tools/utils/models.py

```diff
@@ -356,10 +356,12 @@ def lessThan(self, left_index, right_index):
 
     def filterAcceptsRow(self, source_row, source_parent):
         index = self.sourceModel().index(source_row, 0, source_parent)
+        string_pattern = self.filterRegularExpression().pattern()
         if self._filter_enabled:
             result = self._custom_index_filter(index)
             if result is not None:
-                return result
+                project_name = index.data(PROJECT_NAME_ROLE)
+                return string_pattern in project_name
```

Comment:
Default project has name set to `None`
```suggestion
        if result is not None:
                project_name = index.data(PROJECT_NAME_ROLE)
                if project_name is None:
                    return result
                return string_pattern.lower() in project_name.lower()
```

Reply:
I tested it, it never crashed.

Reply:
Please do it anyway. It's not TrayPublisher's proxy model but general proxy model in utils...